Pipfile

```diff
@@ -3,30 +3,27 @@ url = "https://pypi.python.org/simple"
 verify_ssl = true
 name = "pypi"
 
-
 [packages]
-# Wheels for cryptography until the next release.
+
+# Wheels for cryptography until the next release
 cryptography35_linux = { path='./wheelhouse/cryptography-2.3.dev1-cp35-cp35m-linux_x86_64.whl', markers="python_version >= '3.5' and python_version < '3.6' and platform_system == 'Linux'"}
 cryptography36_linux = { path='./wheelhouse/cryptography-2.3.dev1-cp36-cp36m-linux_x86_64.whl', markers="python_version >= '3.6' and python_version < '3.7' and platform_system == 'Linux'"}
 cryptography36_darwin = { path='./wheelhouse/cryptography-2.3.dev1-cp36-cp36m-macosx_10_13_x86_64.whl', markers="python_version >= '3.6' and python_version < '3.7' and platform_system == 'Darwin'"}
 
-
-rpcudp = {git = "https://github.com/nucypher/rpcudp", ref = "nucypher-depend"}
-kademlia = {git = "https://github.com/nucypher/kademlia", ref = "nucypher-depend"}
-PyNaCl = "*"
-"pysha3" = "*"
-bidict = "*"
-py_ecc = "*"
-SQLAlchemy = "*"
-apistar = "*"
-maya = "*"
-pyumbral = {git = "https://github.com/nucypher/pyumbral.git", ref = "nucypher-depend"}
 requests = "*"
+maya = "*"
 hendrix = "==3.0.0"
 constantSorrow = {git = "https://github.com/nucypher/constantSorrow.git", ref = "nucypher-depend"}
 bytestringSplitter = {git = "https://github.com/nucypher/byteStringSplitter.git", ref = "nucypher-depend"}
-appdirs = "*"
-web3 = "==4.3.0"
+sqlalchemy = "*"
+apistar = "*"
+rpcudp = {git = "https://github.com/nucypher/rpcudp", ref = "nucypher-depend"}
+kademlia = {git = "https://github.com/nucypher/kademlia", ref = "nucypher-depend"}
+pynacl = "*"
+pysha3 = "*"
+py_ecc = "*"
+pyumbral = {git = "https://github.com/nucypher/pyumbral.git", ref = "nucypher-depend"}
```

Comment:
Are we still doing the `nucypher-depend` thing for `pyUmbral`? I thought, we wanted to switch back to `master`?

Reply:
Hmm... I don't (at least immediately) recall agreeing to switch back to `master`, but I do think we're ready (or maybe we wait another 1-2 weeks for this final chips to fall) to move to a depend-on-release format.